packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { switchToBlockType } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { Group, UnGroup } from './icons';
+
+export function ConvertToGroupButton( {
+	onConvertToGroup,
+	onUnCovertFromGroup,
+	isGroupable = false,
+	isUnGroupable = false,
+} ) {
+	return (
+		<Fragment>
+			{ isGroupable && (
+				<MenuItem
+					className="editor-block-settings-menu__control block-editor-block-settings-menu__control"
+					icon={ Group }
+					onClick={ onConvertToGroup }
+				>
+					{ __( 'Group' ) }
```

Comment:
This the verb right? I think we should add context here to remove the ambiguity with the block name. In other languages, these need to be two different strings.

Reply:
> add context here to remove the ambiguity

In English `to group` means to pull things together so it does make sense here. In addition, most other software will use `group` as the term in this context.

If we really need additional context maybe we could try `Group Blocks`/`Ungroup Blocks`?

Reply:
I was actually using "french" and the translation was wrong because it used the block name and not the verb. 

I was actually thinking using https://github.com/WordPress/gutenberg/tree/master/packages/i18n#_x to add the "context" would be enough?

Reply:
Ok great. I've updated with context.